src/management/OrganizationsManager.js

```diff
@@ -0,0 +1,816 @@
+  }
+  if (typeof params.invitation_id !== 'string') {
+    throw new ArgumentError('The invitation ID has to be a string');
+  }
+
+  if (cb && cb instanceof Function) {
+    return this.invitations.delete(params, {}, cb);
+  }
+
+  return this.invitations.delete(params, {});
+};
+
+/**
+ **** Organization Roles Membership
+ */
+
+/**
+ * Get Roles from a Member in a Organization
+ *
+ * @method    getRoles
+ * @memberOf  module:management.OrganizationsManager.prototype
+ *
+ * @example
+ * var params = {id : 'ORGANIZATION_ID', user_id: 'user_id'}
+ * @example <caption>
+ *   This methods takes the organization ID and user ID and returns the roles
+ * </caption>
+ *
+ * management.organizations.getRoles( {id : 'ORGANIZATION_ID', user_id: 'user_id'}, function (err, invite) {
+ *   console.log(invite);
+ * });
+ *
+ * @param   {String}    params.id           ID of the Organization.
+ * @param   {String}    params.user_id      ID of the user.
+ * @param   {Function}  [cb]                Callback function.
+ *
+ * @return  {Promise|undefined}
+ */
+OrganizationsManager.prototype.getMemberRoles = function(params, callback) {
+  return this.roles.getAll(params, callback);
+};
+
+/**
+ * Add a Role to a Member in an organization
+ *
+ * @method    addRoles
+ * @memberOf  module:management.OrganizationsManager.prototype
+ *
+ * @example
+ * var params =  {id : 'ORGANIZATION_ID', user_id: 'user_id'};
+ * var data = {}
+ *
+ * management.organizations.addRoles(params, data, function (err) {
+ *   if (err) {
+ *     // Handle error.
+ *   }
+ * });
+ *
+ * @param   {String}    params.id             ID of the Organization.
+ * @param   {String}    params.user_id        ID of the user.
+ * @param   {Array}     data                  Array of Role IDs
+ * @param   {Function}  [cb]                  Callback function.
+ *
+ * @return  {Promise|undefined}
+ */
+OrganizationsManager.prototype.addMemberRoles = function(params, data, cb) {
+  data = data || [];
+  params = params || {};
+
+  if (!params.id) {
+    throw new ArgumentError('The organization ID passed in params cannot be null or undefined');
+  }
+  if (typeof params.id !== 'string') {
+    throw new ArgumentError('The organization ID has to be a string');
+  }
+
+  if (!params.user_id) {
+    throw new ArgumentError('The user ID passed in params cannot be null or undefined');
+  }
+  if (typeof params.user_id !== 'string') {
+    throw new ArgumentError('The user ID has to be a string');
+  }
+
+  if (cb && cb instanceof Function) {
+    return this.roles.create(params, data, cb);
+  }
+
+  return this.roles.create(params, data);
+};
+
+/**
+ * Remove Roles from a Member of an organization
+ *
+ * @method    removeRoles
+ * @memberOf  module:management.OrganizationsManager.prototype
+ *
+ * @example
+ * var params =  { id :'ORGANIZATION_ID', user_id: 'USER_ID };
+ *
+ * management.organizations.removeRoles(params, function (err) {
```

Comment:
This accepts a second parameter, `data`, which is not included in the documentation.